features/exploredash/src/main/java/org/dash/wallet/features/exploredash/network/service/DashDirectServicesApi.kt

```diff
@@ -30,4 +32,9 @@ interface DashDirectServicesApi {
         @Header(DashDirectConstants.EMAIL) email: String,
         @Body purchaseGiftCardRequest: PurchaseGiftCardRequest
     ): PurchaseGiftCardResponse?
+    @POST("GetMerchantById")
+    suspend fun getMerchantById(
+        //@Header(DashDirectConstants.API_KEY) appKey: String,
+        @Body getMerchantByIdRequest: GetMerchantByIdRequest
+    ): GetMerchantByIdResponse?
```

Comment:
This is a guess what the parameters for this API should be.  The Authentication token appears to be included in the headers.